packages/devtools_app/lib/src/screens/profiler/cpu_profiler.dart

```diff
@@ -365,224 +367,62 @@ class _CpuProfilerState extends State<CpuProfiler>
   }
 }
 
-class DisplayTreeGuidelinesToggle extends StatelessWidget {
-  const DisplayTreeGuidelinesToggle();
+// TODO(kenz): one improvement we could make on this is to show the denominator
+// for filtered profiles (e.g. 'Sample count: 10/14), or to at least show the
```

Comment:
review all the additions from here to the bottom of the file as new code - not a modification of code on the left.